lib/ansible/modules/cloud/vmware/vmware_vmotion.py

```diff
@@ -271,7 +271,8 @@ def migrate_vm(self):
         Migrate virtual machine and return the task.
         """
         relocate_spec = vim.vm.RelocateSpec(host=self.host_object,
-                                            datastore=self.datastore_object)
+                                            datastore=self.datastore_object,
+                                            pool=self.host_object.parent.resourcePool)
```

Comment:
Can we make this configurable ? like `destination_resource_pool` so that even if the destination host has resource pool different than default 'Resources' it can be used.

Reply:
Done the changes. Please review